### PR TITLE
freshdesk-import: rename to import, add metadata to ZT members

### LINKED ISSUE #18: Import HoloPort metadata into ZeroTier Central
It would be great to have meaningful name/description set to individual ZeroTier devices. #19 maps agent IDs to names and emails to descriptions, and once it's merged, this will happen on its own for new registrations.

We should import this data from Freshdesk to ZeroTier Central so that we would also keep track of old registrations the same way.

### PR DESCRIPTION
**Pull request by yegortimoshenko**
Now import tool populates ZeroTier Central members with name/description metadata that maps onto Holochain agent IDs and email addresses respectively. Resolves #18.

### CODE REVIEW

**Comment by yegortimoshenko** [comment]
cc @peeech who reviewed the first iteration of this tool yesterday.

File: import/holo_auth_import/main.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+from requests.adapters import HTTPAdapter
+import json
+import logging
+import os
+import requests
+import time
+
+
+CLOUDFLARE_ACCOUNT_ID = '18ff2b4e6205b938652998cfca0d8cff'
+CLOUDFLARE_AGENT_ID_TO_IPV4_NAMESPACE_ID = 'e3de244cdc5241ce8b68096a3fab78ff'
```
Comment (peeech) [inline import/holo_auth_import/main.py]: Out of curiosity how do you populate this KV store? `zerotier_network_update_member()` will be called on an intersection of data from many sets, including `CLOUDFLARE_AGENT_ID_TO_IPV4_NAMESPACE_ID` so I want to make sure I understand this one.

Comment (yegortimoshenko) [reply import/holo_auth_import/main.py]: This KV store is populated by Holo Router:

```js
  await AGENT_ID_TO_IPV4.put(payload.holochain_agent_id.toLowerCase(), zerotierPayload.config.ipAssignments[0])
```

Comment (yegortimoshenko) [reply import/holo_auth_import/main.py]: We're only doing this import once, and Holo Router agent ID to IPv4 address mapping is the only way to infer this data. For future entries, Holo Auth will add metadata on its own: #19.

**Pull request merged**